File: app/src/main/java/podtest/jiki/se/podtest/model/Liveaudio.java
package podtest.jiki.se.podtest.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Liveaudio {

    @SerializedName("id")
    @Expose
    private Integer id;
    @SerializedName("url")
    @Expose
    private String url;
    @SerializedName("statkey")
    @Expose
    private String statkey;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public String getStatkey() {
        return statkey;
    }

    public void setStatkey(String statkey) {
        this.statkey = statkey;
    }

}

File: app/src/main/java/se/asapehrsson/podtest/data/Liveaudio.kt
package se.asapehrsson.podtest.data

import com.google.gson.annotations.SerializedName

data class Liveaudio(
        @SerializedName("id") val id: Int? = null,
        @SerializedName("url") val url: String? = null,
        @SerializedName("statkey") val statkey: String? = null
)